api/src/models/pitch.ts

```diff
@@ -1,76 +1,63 @@
-import mongoose, { Document, Schema } from 'mongoose';
-import { IUser } from './user';
+import mongoose, { Schema, Document } from 'mongoose';
+import { IPitch } from '../types/index';
 
 import {
   interestsEnum,
   pitchStatusEnum,
   assignmentStatusEnum,
 } from '../utils/enums';
 
-export interface IPitch extends Document<any> {
-  _id: string;
-  name: string;
-  pitchAuthor: mongoose.Types.ObjectId;
-  pitchStatus: string;
-  assignmentStatus: string;
-  assignmentGoogleDocLink: string;
-  assignmentContributors: [IUser];
-  topic: string;
-  currentWriters: number;
-  targetWriters: number;
-  currentEditors: number;
-  targetEditors: number;
-  currentData: number;
-  targetData: number;
-  currentVisuals: number;
-  targetVisuals: number;
-  currentIllustration: number;
-  targetIllustration: number;
-  currentPhotography: number;
-  targetPhotography: number;
-  currentFactChecking: number;
-  targetFactChecking: number;
-  currentRadio: number;
-  targetRadio: number;
-  currentLayout: number;
-  targetLayout: number;
-  approvedBy: IUser;
-  similarStories: [string];
-  deadline: Date;
-}
-
 /**
  * Mongoose Schema to represent a Pitch at South Side Weekly
  */
 const Pitch = new mongoose.Schema({
   name: { type: String, default: null, required: true },
   pitchAuthor: { type: Schema.Types.ObjectId, ref: 'User', required: true },
-  pitchStatus: { type: pitchStatusEnum, default: pitchStatusEnum.NONE },
+  pitchStatus: {
+    type: String,
```

Comment:
I notice that you are typing as `String` instead of `string`. Is there a reason we need a String object instead of a standard string primitive?